src/Components/ArtistFilter/Artworks.tsx

```diff
@@ -0,0 +1,147 @@
+import * as React from "react"
+import {
+  ConnectionData,
+  createPaginationContainer,
+  graphql,
+  RelayPaginationProp,
+} from "react-relay"
+import { Subscribe } from "unstated"
+
+import { FilterState } from "./state"
+import ArtworkGrid from "../ArtworkGrid"
+
+interface Props extends RelayProps {
```

Comment:
You don’t need to write and `RelayProps` anymore. Instead import the TS typings emitted by our relay-compiler TS plugin, in this case you should import from [this file](https://github.com/artsy/reaction/blob/d31a2db8ff9bc55e1eaba8d88ad8ccbf556f27f5/src/__generated__/Artworks_filtered_artworks.graphql.ts):

```ts
import { Artworks_filtered_artworks } from "../../__generated__/Artworks_filtered_artworks.graphql"

interface Props {
  filtered_artworks: Artworks_filtered_artworks
}
```